en/simulation/hitl.md

```diff
@@ -1,101 +1,133 @@
+## HITL-Compatible Airframes {#compatible_airframe}
 
-PX4 supports HITL for multicopters \(using jMAVSim or Gazebo\) and fixed wing \(using X-Plane demo or full version or Gazebo\). Generally any compatible airframe can be put into HITL mode. Compatible airframes are right now:
+The current list of compatible airframes vs Simulators is:
 
-* X-frame multicopters \(autostart IDs 4001, 4011\)
-* Standard AERT planes
-* HITL X-frame multicopter \(autostart ID 1001\)
-* HITL AERT plane \(autostart ID 1000\)
+Airframe | `SYS_AUTOSTART` | X-Plane | Gazebo | jMAVSim
+---|---|---
+<a href="../airframes/airframe_reference.md#plane_simulation_(plane)_hilstar_(xplane)">HILStar (X-Plane)</a> | 1000 | Y | |
+<a href="../airframes/airframe_reference.md#copter_simulation_(copter)_hil_quadcopter_x">HIL Quadcopter X</a> | 1001 | | Y | Y
+[Standard planes](../airframes/airframe_reference.md#plane_standard_plane_standard_plane) | 2100 | Y | |
+[Generic Quadrotor x](../airframes/airframe_reference.md#copter_quadrotor_x_generic_quadrotor_x) copter | 4001 | | Y | Y
+[DJI Flame Wheel f450](../airframes/airframe_reference.md#copter_quadrotor_x_dji_flame_wheel_f450) | 4011 | | Y | Y
 
-The screenshot below shows the simulation airframe selected. Select HILStar for XPlane Simulator and fixed wing and Quadrotor X for jMAVSim or Gazebo.
 
-![](../../assets/gcs/qgc_hil_config.png)
+## HITL Simulation Environment {#simulation_environment}
 
-For Gazebo, set the `SYS_COMPANION` parameter to `921600`. 
+With Hardware-in-the-Loop (HITL) simulation the normal PX firmware is run on real hardware. *QGroundControl* is connected to the physical hardware over USB and acts as a gateway to forward data between the simulator running on a development computer, PX4, and any offboard API.
 
-Under the `General` tab of the settings menu, uncheck all AutoConnect boxes except for UDP. 
+The diagram below shows a typical HITL simulation environment:
+* A HITL configuration is selected (via *QGroundControl*) that doesn't start any real sensors.
+* *QGroundControl* is connected to the flight controller via the USB.
+* *QGroundControl* is connected to the simulator and offboard API via UDP.
+* A serial connection is used to connect Joystick/Gamepad hardware via *QGroundControl*.
 
-![](../../assets/sim/QGC_AutoConnect.png)
+![HITL setup](../../assets/simulation/px4_hitl_overview.png)
 
-Once configuration in QGroundControl is complete, close QGroundControl and disconnect the autopilot from the computer. 
+> **Tip** For more information see: [Simulation](../simulation/README.md).
 
-## Configure Joystick and Failsafe \(Optional\)
 
-These parameter are optional and allow to run HITL without a radio remote controller connected. Use the search bar on the top of the parameter editor to bring them up.
+## HITL vs SITL
```

Comment:
@dagar Would be good if you could check this section. I would have preferred something that says a bit more precisely the relative "accuracy" but I think this at least removes the impression that HITL is a "perfect" autopilot simulation.